### DISCUSSION TITLE
An exception occurred in the driver: could not find driver

### DISCUSSION THREAD

**Discussion by dupuishugo80** [dunglas/symfony-docker#364] [Q&A] [answered]
Hi,

I just installed the container and installed some libraries ,and when I wanted to create a database I have this error

" An exception occurred in the driver: could not find driver "

Thanks :)

**Comment by maxhelias** [comment, marked as answer]
Hi!
You need to install the corresponding driver on the php container. You can use this lib https://github.com/mlocati/docker-php-extension-installer provided in this project
